ROMFS/px4fmu_common/init.d-posix/rcS

```diff
@@ -191,15 +191,15 @@ fi
 
 # Adapt timeout parameters if simulation runs faster or slower than realtime.
 if [ ! -z $PX4_SIM_SPEED_FACTOR ]; then
-	COM_DL_LOSS_T_LONGER=$((PX4_SIM_SPEED_FACTOR * 2))
```

Comment:
Why did you re-introduce BC here? That's missing in some systems.

Reply:
See https://github.com/PX4/Firmware/pull/13867.